CommonSelectors.py

```diff
@@ -389,83 +396,45 @@ def TrueJetFlavors(events, params, **kwargs):
 )
 
 
-ll_2j = Cut(
-    name = 'll_2j',
-    function=TwoLepTwoJets,
-    params={"lep_flav": "both",
-            "pt_dilep": 60,
-            "mll": {'low': 50, 'high': 400}
-            }
-)
-mumu_2j = Cut(
-    name = 'mumu_2j',
-    function=TwoLepTwoJets,
-    params={"lep_flav": "mu",
-            "pt_dilep": 60,
-            "mll": {'low': 50, 'high': 400}
-        }
-)
-ee_2j = Cut(
-    name = 'ee_2j',
-    function=TwoLepTwoJets,
-    params={"lep_flav": "el",
-            "pt_dilep": 60,
-            "mll": {'low': 50, 'high': 400}
-        }
-)
-
-Zll_2j = Cut(
-    name = 'Zll_2j',
-    function=TwoLepTwoJets,
-    params={"lep_flav": "both",
-            "pt_dilep": 60,
-            "mll": {'low': 75, 'high': 115}
-            }
-)
-
-
-dilep_pt60to150 = Cut(
-    name="dilep_pt60to150",
-    function=DiLeptonPtCut,
-    params={
-	"ptll": {'low': 60, 'high': 150}
-    },
-)
+def ll_2j(lep_flav='both'):
+    return Cut(
+        name = 'll_2j_'+lep_flav,
+        function=TwoLepTwoJets,
+        params={"lep_flav": lep_flav,
+                "pt_dilep": 60,
+                "mll": {'low': 50, 'high': 400}
+                }
+    )
 
-dilep_pt150to2000 = Cut(
-    name="dilep_pt150to2000",
-    function=DiLeptonPtCut,
-    params={
-	"ptll": {'low': 150, 'high': 2000}
-    },
-)
+def Zll_2j(lep_flav='both'):
+    return  Cut(
+        name = 'Zll_2j',
+        function=TwoLepTwoJets,
+        params={"lep_flav": lep_flav,
+                "pt_dilep": 60,
+                "mll": {'low': 75, 'high': 115}
+                }
+    )
 
-Zmumu_2j = Cut(
-    name = 'Zmumu_2j',
-    function=TwoLepTwoJets,
-    params={"lep_flav": "mu",
-            "pt_dilep": 60,
-            "mll": {'low': 75, 'high': 115}
-        }
-)
-Zee_2j = Cut(
-    name = 'Zee_2j',
-    function=TwoLepTwoJets,
-    params={"lep_flav": "el",
-            "pt_dilep": 60,
-            "mll": {'low': 75, 'high': 115}
-        }
-)
 
+def dilep_pt(pt_min=60, pt_max=2000):
+    return Cut(
+        name="dilep_pt_cut",
+        function=DiLeptonPtCut,
+        params={
+	    "ptll": {'low': pt_min, 'high': pt_max}
+        },
+    )
 
 # Cuts for ttbar control region
-ll_antiZ_4j = Cut(
-    name = 'll_antiZ_4j',
-    function=AntiZFourJets,
-    params={"lep_flav": "both",
-            "pt_dilep": 60,
-            "mll": {'low': 75, 'high': 120}
-            }
-)
+def ll_antiZ_4j(lep_flav='both'):
+    return  Cut(
+        name = 'll_antiZ_4j',
+        function=AntiZFourJets,
+        params={"lep_flav": lep_flav,
+                "pt_dilep": 60,
+                "mll": {'low': 75, 'high': 120}
+                }
+    )
```

Comment:
Looks fine to me. Are you going to transfer 1L and 0L channels as well?

Reply:
Yes, for 1L